docker-compose.yml

```diff
@@ -49,9 +49,9 @@ services:
       - rabbitmq_data:/var/lib/rabbitmq
     healthcheck:
       test: ["CMD", "rabbitmqctl", "status"]
-      interval: 30s
+      interval: 10s
       timeout: 30s
-      retries: 3
+      retries: 9
```

Comment:
This conflicts with the changes I just merged: https://github.com/OpenRailAssociation/osrd/pull/9166/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3
(same for other compose files was done in my PR).
Changes introduced aimed at being consistent on all deps of OSRD's stack (so after a rebase I would say this change is not necessary anymore).